e2e/dr_test.go

```diff
@@ -31,7 +32,12 @@ func TestDR(dt *testing.T) {
 	}
 
 	t.Cleanup(func() {
-		if err := util.EnsureChannelDeleted(&Ctx); err != nil {
+		deadline := time.Now().Add(1 * time.Minute)
+		timedCtx, cancel := Ctx.WithDeadline(deadline)
```

Comment:
Will be simpler as:

    timedCtx, cancel := Ctx.WithDeadline(1 * time.Minute)